apps/client/src/components/qna/QnaListBox/index.tsx

```diff
@@ -20,56 +30,61 @@ const QnaListBox = () => {
     <StyledQnaListBox>
       {QNA_SORT.map((qna) => (
         <Text
-          key={qna.id}
           fontType="p1"
           onClick={() => setSelectedQna(qna.id)}
           style={{
             marginRight: '7px',
             cursor: 'pointer',
             fontWeight: selectedQna === qna.id ? 'bold' : 'normal',
           }}
+          key={qna.id}
         >
           {qna.title}
         </Text>
       ))}
       <StyledQnaBoxWrapper>
-        {QNA.map((qna) => (
-          <StyledQnaBox key={qna.id} onClick={() => handleGoDetailPage(qna.id)}>
-            <Flex direction="column">
-              <StyledPopularQna>
-                <QnaCategory questionType={Qna.CONCERN} />
-                <Text fontType="h4">{qna.title}</Text>
-                <StyledQnaContent>{qna.detailContent}</StyledQnaContent>
-                <Flex justify="space-between" style={{ width: '100%' }}>
-                  <Stack spacing={12} direction="horizontal">
-                    <Text fontType="body2">{qna.name}</Text>
-                    <Text fontType="body2">6일 전</Text>
-                  </Stack>
-                  <Stack spacing={12} direction="horizontal">
-                    <Stack direction="horizontal" align="center" spacing={3}>
-                      <IconHeartRegular width={16} height={16} color={colors.black} />
-                      <Text fontType="body2" style={{ marginTop: '2px' }}>
-                        {qna.heart}
-                      </Text>
+        {qnaList.length > 0 ? (
+          qnaList.map((qna) => (
+            <StyledQnaBox onClick={() => handleGoDetailPage(qna.id)} key={qna.id}>
+              <Flex direction="column">
+                <StyledPopularQna>
+                  <QnaCategory questionType={qna.category as unknown as Qna} />
```

Comment:
```suggestion
                  <QnaCategory questionType={qna.category as Qna} />
```

qnaList에 근데 타입 지정 안해놓으신건가용?